panel/src/components/Forms/Fieldset.vue

```diff
@@ -4,7 +4,7 @@
 			<template v-for="(field, fieldName) in fields">
 				<k-column
 					v-if="$helper.field.isVisible(field, value)"
-					:key="field.signature"
+					:key="field.name"
```

Comment:
`Question:` Reading this, is there a difference between `fieldName` and `field.name` here?

Reply:
Nope. Could totally go with fieldName instead.

Reply:
I'd probably do it the other way around and maybe shift to `field.name` instead of the key all together. But doesn't need to be in this PR.

Reply:
I've fixed it.